lib/src/common/repos_service.dart

```diff
@@ -208,86 +247,192 @@ class RepositoriesService extends Service {
         (json) => Collaborator.fromJson(json),
       );
 
-  Future<bool> isCollaborator(RepositorySlug slug, String user) {
-    return _github
-        .request("GET", "/repos/${slug.fullName}/collaborators/$user")
-        .then((response) {
-      return response.statusCode == 204;
-    });
+  Future<void> isCollaborator(RepositorySlug slug, String user) async {
```

Comment:
This one confused me at first since usual Dart convention of a method starting with `is` is to return a `bool`.
Looking at this method, I would assume it would resolve the future with a bool whether the user is or isn't a collaborator. If there were a network error or otherwise, it would throw. 
How does this method behave when the user isn't a collaborator?

Reply:
I made some changes so it returns a bool as it did before.